src/vscode-dts/vscode.proposed.telemetryLogger.d.ts

```diff
@@ -57,13 +58,6 @@ declare module 'vscode' {
 		 */
 		logEvent(eventName: string, data?: Record<string, any>): void;
 
-		/**
-		 * User-defined function which logs an error, used within the TelemetryLogger
-		 * @param exception The exception being logged
-		 * @param data Any additional data to be collected with the exception
-		 */
-		logException(exception: Error, data?: Record<string, any>): void;
-
 		/**
```

Comment:
I am unsure about this. IMO being aware of errors and exceptions is a crucial part of this API and now you need to spec some kind of format how an error is fit into a record (instead of the `Error` type)

Reply:
We can discuss this during the API sync. This issue is there is really only app insights that has `trackException` and it's not used often for logging errors. All telemetry libraries are guaranteed to have a `logEvent` but I don't think we need a special case for log exception. The returned logger will still allow you just to pass an `Error` object but we will translate it for you.